techniques/html/H73.html

```diff
@@ -10,31 +10,31 @@
             when there are several sets of row or column headers, or when there are multiple groups
-            of columns or rows). The <code class="att">summary</code> may also be helpful for simple data tables
+            of columns or rows). The <code>summary</code> may also be helpful for simple data tables
             that contain many columns or rows of data.</p>
-      <p>The <code class="att">summary</code> attribute may be used whether or not the table includes a
-              <code class="el">caption</code> element. If both are used, the <code class="att">summary</code> should not
-            duplicate the <code class="el">caption</code>.</p>
-      <div class="note"><p>In HTML5 the <code class="att">summary</code> attribute is obsolete. Assistive technologies may or may not continue to support the attribute. Authors should consider alternatives and only use with caution.</p></div>
+      <p>The <code>summary</code> attribute may be used whether or not the table includes a
+              <code>caption</code> element. If both are used, the <code>summary</code> should not
+            duplicate the <code>caption</code>.</p>
+      <div class="note"><p>In HTML5 the <code>summary</code> attribute is obsolete. Assistive technologies may or may not continue to support the attribute. Authors should consider alternatives and only use with caution.</p></div>
       <div class="note">
          <p> Although WCAG 2 does not prohibit the use of layout tables, CSS-based layouts are
-               recommended in order to retain the defined semantic meaning of the HTML <code class="el">table</code>
+               recommended in order to retain the defined semantic meaning of the HTML <code>table</code>
                elements and to conform to the coding practice of separating presentation from content.
-               However, if a layout table is used, then the <code class="att">summary</code> attribute is not used or
+               However, if a layout table is used, then the <code>summary</code> attribute is not used or
                is null. The purpose of a layout table is simply to control the placement of content;
-               the table itself is “transparent" to the user. A <code class="att">summary</code> would “break" this
-               transparency by calling attention to the table. A null <code class="att">summary</code>
-                 (<code class="att">summary</code>="") on layout tables is acceptable. See <a href="https://www.w3.org/WAI/WCAG21/Techniques/failures/F46">F46</a> for details.</p>
+               the table itself is “transparent" to the user. A <code>summary</code> would “break" this
+               transparency by calling attention to the table. A null <code>summary</code>
+                 (<code>summary</code>="") on layout tables is acceptable. See <a href="https://www.w3.org/WAI/WCAG21/Techniques/failures/F46">F46</a> for details.</p>
```

Comment:
```suggestion
                 (<code>summary=""</code>) on layout tables is acceptable. See <a href="https://www.w3.org/WAI/WCAG21/Techniques/failures/F46">F46</a> for details.</p>
```

and could do with highlighting